apps/E2E/src/Callout/pages/CalloutPageObject.win.ts

```diff
@@ -14,32 +14,20 @@ class CalloutPageObject extends BasePage {
     return await (await this._primaryComponent).isDisplayed();
   }
 
-  /* OVERRIDE: This must scroll to the button that opens the callout, not the callout (since it's not visible on load.) */
-  async scrollToTestElement(): Promise<void> {
-    // on win32, adding a blank value to an element automatically scrolls to it
-    await (await this._buttonToOpenCallout).addValue('');
-  }
-
-  async openCallout(): Promise<void> {
+  // This both opens and waits for it to go in view
+  async openCalloutAndWaitForLoad(): Promise<void> {
     await (await this._buttonToOpenCallout).click();
```

Comment:
I think we should check if it's already open (doesn't hurt). I could see it being a headache for someone if they're calling this and it's already open (then this function will close it and the `wait()` will fail